jupyter-book/spatial/neighborhood.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    )  # <code>method</code> compute the hierarchical clustering dendrogram for the heatmap
</pre><p> </p><p>Weird comment + position</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>